tileview/src/main/java/com/qozix/tileview/tiles/Tile.java

```diff
@@ -167,13 +169,14 @@ public void setState( State state ) {
     mState = state;
   }
 
-  public void execute( TileRenderPoolExecutor tileRenderPoolExecutor ) {
+  public void execute( TileRenderPoolExecutor tileRenderPoolExecutor, BitmapRecycler recycler ) {
```

Comment:
make sure this is non-breaking...  provide a overloaded signature that matches the original

```
public void execute( TileRenderPoolExecutor tileRenderPoolExecutor ) {
  execute( tileRenderPoolExecutor, null );
}
```
And of course make sure that passing a `null` here won't break anything